GPT/gpt.talon

```diff
@@ -10,8 +10,8 @@ model <user.modelPrompt> [{user.modelSource}] [{user.modelDestination}]:
     result = user.gpt_apply_prompt(modelPrompt, text)
     user.gpt_insert_response(result, modelDestination or "")
 
-# Select the last GPT response so you can edit it further
-model take response: user.gpt_select_last()
```

Comment:
We are getting rid of this? This doesn't seem contradictory to the new addition

Reply:
Oops. That looks like a mistake. I'm afk but will put that back if you don't beat me to it.

Reply:
Reverted that accidental deletion